guestbook/php-redis/guestbook.php

```diff
@@ -19,9 +19,7 @@
 error_reporting(E_ALL);
 ini_set('display_errors', 1);
 
-require 'Predis/Autoloader.php';
-
-Predis\Autoloader::register();
```

Comment:
I am able to remove this `Predis\Autoloader::register();` line because Composer's `vendor/autoload.php` already takes care of that.